src/kbase/auth/client.py

```diff
@@ -0,0 +1,6 @@
+"""
+The aync and sync versions of the KBase Auth Client.
+"""
+
+from kbase.auth._async.client import AsyncClient as AsyncKBaseAuthClient  # @UnusedImport
+from kbase.auth._sync.client import Client as KBaseAuthClient  # @UnusedImport
```

Comment:
what tool/IDE is the `@UnusedImport` annotation for? ruff uses [`unused-import` or `F401`](https://docs.astral.sh/ruff/rules/unused-import/).